src/openforms/registrations/contrib/objects_api/templates/registrations/contrib/objects_api/content_json.txt

```diff
@@ -0,0 +1,16 @@
+{% load registrations.contrib.objects_api.json_summary capture_tags %}
+{% capture as tt_openblock silent %}{% templatetag openblock %}{% endcapture %}
+{% capture as tt_closeblock silent %}{% templatetag closeblock %}{% endcapture %}
+{% capture as tt_openvariable silent %}{% templatetag openvariable %}{% endcapture %}
+{% capture as tt_closevariable silent %}{% templatetag closevariable %}{% endcapture %}
+{% autoescape off %}
```

Comment:
you can remove this and disable auto-escaping in the `plugin.py` as mentioned before